app/util/color.ts

```diff
@@ -12,6 +12,20 @@ const MATERIAL_CHART_COLORS = [
   "#3F51B5", // indigo-500
 ];
 
+/**
+ * A subset of material color pairs. Each pair includes a dark version and
+ * a light version.
+ */
+const MATERIAL_CHART_COLORS_DARK_AND_LIGHT = [
+  ["#4CAF50", "#A5D6A7"], // green-200
+  ["#03A9F4", "#90CAF9"], // blue-200
+  ["#FF9800", "#FFCC80"], // orange-200
+  ["#9C27B0", "#CE93D8"], // purple-200
+  ["#F44336", "#EF9A9A"], // red-200
+  ["#009688", "#80CBC4"], // teal-200
+  ["#3F51B5", "#9FA8DA"], // indigo-200
+];
```

Comment:
nit: instead of modeling these as pairs, it might be slightly simpler/more generally useful to have this be a list of just the light colors (not dark colors), then call both `getChartColor()` + `getLightChartColor()`